### ISSUE TITLE
Replace PropertySet with Property

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/forces-and-motion-basics#226]
From phetsims/axon#102, PropertySet is deprecated and should be replaced with Property.

**Comment by jessegreenberg** [comment]
In #235, we will have a new RC for Forces and Motion Basics.  If we do this quickly now, the replacement will be tested in the next RC.

**Comment by jessegreenberg** [comment]
Actually, this conversion could definitely have an impact on phetsims/QA#12, and may make it difficult to cherry-pick changes into the release branch 2.3-phetio.  Ill coordinate with @zepumph about this, either waiting until 2.3-phetio is deployed or if we decide 2.3-phetio needs master SHAs.

**Comment by zepumph** [comment]
Thanks @jessegreenberg. I hope to have #232 figured out before I leave Friday, so that it won't put this on hold for too long.

**Comment by jessegreenberg** [comment]
Thanks @zepumph.  In #235 we decided to take both 2.3 and 2.3-phetio from master so I will proceed with this issue now.

**Comment by jessegreenberg** [comment]
One down, 5 to go.

**Comment by jessegreenberg** [comment]
I was concerned about breaking phetio features during this refactor.  In addition to sanity checking wrappers, @zepumph recommended to compare ids in the instance proxies wrapper before and after the change. I have a list of id's before changes related to this, and will compare once refactor is done.

**Comment by jessegreenberg** [comment]
PropertySet was removed from this sim.  Comparing the phetio id's before and after the refactor is showing no changes:

<img width="1439" alt="screen shot 2017-08-21 at 4 42 53 pm" src="https://user-images.githubusercontent.com/6396244/29541213-e7a482b0-868f-11e7-8e7d-d2e1a215d5a8.png">

This sim has a lot of Properties, and it was using Events and convenience functions of PropertySet, so I want to do some testing to make sure that things didn't break.

**Comment by jessegreenberg** [comment]
PropertySet has been removed. I have been spot checking the sim, trying to break things after the refactor but features seem stable.  Ill remove references to Property.preventGetSet and close.
